bi_sql_editor/models/bi_sql_view.py

```diff
@@ -218,10 +226,9 @@ def button_open_view(self):
         return {
             'type': 'ir.actions.act_window',
             'res_model': self.model_id.model,
-            'view_id': self.graph_view_id.id,
             'search_view_id': self.search_view_id.id,
-            'view_type': 'graph',
-            'view_mode': 'graph',
+            'view_type': 'form',
```

Comment:
No blocking from me but 'view_type' is no more required/used: it works without it